nb/updatecenters/src/org/netbeans/modules/updatecenters/resources/Bundle.properties

```diff
@@ -33,3 +33,7 @@ URL_Distribution=https://netbeans.apache.org/nb/updates/11.1/updates.xml.gz?{$ne
 URL_PluginPortal=https://netbeans.apache.org/nb/plugins/11.1/catalog.xml.gz
```

Comment:
```suggestion
URL_PluginPortal=http://plugins.netbeans.org/nbpluginportal/updates/11.0/catalog.xml.gz
```

Reply:
That is already where it goes to at the moment - see https://github.com/apache/netbeans-website/blob/master/netbeans.apache.org/src/content/.htaccess#L9  There was a discussion a while back about trying to keep all IDE inbound links under one root where possible - eg. https://netbeans.apache.org/nb/